src/c++/perf_analyzer/docs/measurements_metrics.md

```diff
@@ -0,0 +1,128 @@
+<!--
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Measurement Modes
+
+Currently, Perf Analyzer has 2 measurement modes.
+
+## Time Windows
+
+When using time windows measurement mode
+([`--measurement-mode=time_windows`](cli.md#--measurement-mode=[time_windows|count_windows])),
+Perf Analyzer will count how many requests have completed during a window of
+duration `X` (in milliseconds, via `--measurement-interval=X`, default is
+`5000`). This is the default measurement mode.
+
+## Count Windows
+
+When using count windows measurement mode
+([`--measurement-mode=count_windows`](cli.md#--measurement-mode=[time_windows|count_windows])),
+Perf Analyzer will start the window duration at 1 second and potentially
+dynamically increase it until `X` requests have completed (via
+`--measurement-request-count=X`, default is `50`).
+
```

Comment:
Can you add a section `# Metrics` and put the contents from [this](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/README.md#understanding-the-output) here?

Reply:
done